src/onboarding/ha-onboarding.js

```diff
@@ -0,0 +1,124 @@
+import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-button/paper-button.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import hassCallApi from '../util/hass-call-api.js';
+
+const callApi = (method, path, data) => hassCallApi('', {}, method, path, data);
+
+class HaOnboarding extends PolymerElement {
+  static get template() {
+    return html`
+    <style include="iron-flex iron-positioning"></style>
+    <style>
+      .layout {
+        padding-top: 20px;
+      }
+
+      .error {
+        color: red;
+        font-weight: bold;
+      }
+    </style>
+    <div class="layout vertical center fit">
+      <img src="/static/icons/favicon-192x192.png" height="192">
+
+      <p>Create your user account.</p>
```

Comment:
Is this view for the admin/owner account only? Maybe we should mention it then

Reply:
Yes, this view is only shown once during initial setup of Home Assisatant. Think of it like setting up a new Windows or MacOS installation.